corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,4 +1,4 @@
-DOMAIN_INDEX = "hqdomains_391fc13ad99b11e48da43c970e7be43b"
+DOMAIN_INDEX = "hqdomains_20150403_1448"
```

Comment:
Does this affect the calculation of a diff though? or is the delta calculation done on the fly with what's in source control (live) and what changes it calculates from the Case object?